ping_one/README.md

```diff
@@ -1,42 +1,61 @@
-# Agent Check: ping_one
+# PingOne Integration for Datadog
 
 ## Overview
 
-This check monitors [PingOne][1].
+[PingOne][1] is an identity-as-a-service (IDaaS) offering by Ping Identity. It's a cloud-based identity platform that provides a range of services related to identity and access management (IAM), including single sign-on (SSO), multi-factor authentication (MFA), user management, and more.
+
+The PingOne integration collects Audit logs and sends them to Datadog. Using the out-of-the-box logs pipeline, the logs are parsed and enriched for easy searching and analysis. This integration includes several dashboards visualizing total Audit events, total successful/failed login attempts, total successful/failed kerberos login attempts, and more.
 
 ## Setup
 
-### Installation
+### Configuration
+
+#### PingOne Configuration
 
-The PingOne check is included in the [Datadog Agent][2] package.
-No additional installation is needed on your server.
+1. Login to [**PingOne**][2] with your credentials.
+2. From the navigation sidebar, expand the **Applications** section and select **Applications**.
+3. Click + to begin creating a new application.
+4. Enter an **Application Name**.
+5. Select **Worker** as the application type.
+6. On the application flyout, ensure that the toggle switch in the header is activated, in order to enable the application.
+7. Select the **Roles** tab of the application flyout.
+8. Click the **Grant** Roles button.
```

Comment:
```suggestion
8. Click the **Grant Roles** button.
```

Or if "Roles" is not part of the button, it probably shouldn't be capitalized, but I'm not sure without seeing a screenshot.

Reply:
![Screenshot 2024-07-09 115200](https://github.com/DataDog/integrations-core/assets/95739270/d635991d-6632-4405-894b-51b42fe5c9c9)
It is a part of the button